node/overseer/src/lib.rs

```diff
@@ -112,7 +113,7 @@ pub use orchestra::{
 
 /// Store 2 days worth of blocks, not accounting for forks,
 /// in the LRU cache. Assumes a 6-second block time.
-pub const KNOWN_LEAVES_CACHE_SIZE: usize = 2 * 24 * 3600 / 6;
+pub const KNOWN_LEAVES_CACHE_SIZE: Option<NonZeroUsize> = NonZeroUsize::new(2 * 24 * 3600 / 6);
```

Comment:
to avoid this pesky expects, it might be nicer to do a match
https://github.com/paritytech/polkadot/blob/30e65cb496f584d461104a17b0c4cb403d029b84/node/network/collator-protocol/src/collator_side/validators_buffer.rs#L46-L49

Reply:
Much better, thanks :+1: